apps/look-and-feel-stories/src/ContentItemDuoAction/ContentItemDuoAction.stories.tsx

```diff
@@ -0,0 +1,50 @@
+import { Meta, StoryObj } from "@storybook/react";
+import bank from "@material-symbols/svg-700/rounded/account_balance_wallet-fill.svg";
+import { ContentItemDuoAction } from "@axa-fr/design-system-apollo-react";
```

Comment:
ici tu devrais plutot importer celui de l&f : 
```suggestion
import { ContentItemDuoAction } from "@axa-fr/design-system-apollo-react/lf";
```